PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -35,13 +35,21 @@
 
 #include "PWGLF/DataModel/LFStrangenessTables.h"
 
+#include "Tools/KFparticle/KFUtilities.h"
 #include "PWGHF/Core/CentralityEstimation.h"
 #include "PWGHF/Core/SelectorCuts.h"
 #include "PWGHF/DataModel/CandidateReconstructionTables.h"
 #include "PWGHF/DataModel/CandidateSelectionTables.h"
 #include "PWGHF/Utils/utilsBfieldCCDB.h"
 #include "PWGHF/Utils/utilsEvSelHf.h"
 
+/// includes KFParticle
+#include "KFParticle.h"
+#include "KFParticleBase.h"
+#include "KFPTrack.h"
+#include "KFPVertex.h"
+#include "KFVertex.h"
```

Comment:
These are external headers. They should be included before O2 headers.

Reply:
Done.